js/app/modules/navigation/topMenu.js

```diff
@@ -43,22 +43,27 @@ const TopMenu = new Module.Class({
     },
 
     _init: function (props={}) {
-        this.hexpand = true;
+
+        props.hexpand = true;
+        props.orientation = Gtk.Orientation.HORIZONTAL;
+        props.spacing = props.spacing | 8;
+
         this.parent(props);
 
         this._banner = this.create_submodule('banner');
         this._menu = this.create_submodule('menu');
 
+        this._banner.hexpand = false;
```

Comment:
You can pass these directly to `create_submodule()` if you like, then they get passed in at construct time:
```js
this._banner = this.create_submodule('banner', {
    hexpand: false,
    valign: Gtk.Align.CENTER,
});
```